BUILD.md

```diff
@@ -191,9 +191,32 @@ See more information on the TensorRT Execution Provider [here](./docs/execution_
 
 Dockerfile instructions are available [here](./dockerfiles#tensorrt)
 
-#### Jetson (ARM64 Builds)
+#### Jetson TX1/TX2/Nano(ARM64 Builds)
 
-See [instructions](https://github.com/microsoft/onnxruntime/issues/2684#issuecomment-568548387) for additional information and tips related to building Onnxruntime with TensorRT Execution Provider on Jetson platforms (TX1/TX2, Nano)
+
+1. ONNX Runtime v1.2.0 or higher requires TensorRT 7 support, make sure Jetson device is upgraded to [JetPack](https://docs.nvidia.com/jetson/jetpack/release-notes/) 4.4 or higher.
```

Comment:
were you able to validate this?
JetPack 4.4 looks to be in preview and seems to depend on TensorRT 7.1 and cuDNN 8.0
AFAIK, we've only been testing TensorRT 7.0 , cuDNN 7.6

Reply:
I tried to upgrade the TX2 device to JetPack 4.4 from JetPack 4.3 thorugh sdkmanager, and encountered many errors. For examples, "Unhandled error when running sudo apt-get -y update" and "dpkg install on target failed". It's possible that JetPack 4.4 is in preview and not stable yet as you mentioned. I will keep following it.

At the same time, I've successfully built and run onnxruntime v1.1.2 with JetPack 4.3 which uses TenssorRT 6. I will revise the doc to describe how to build older onnxruntime with TenssorRT 6 support.

Reply:
Thanks! In addition to the issues you encountered, there will probably need to be some code changes required for ORT to support TensorRT 7.1 properly. 

Btw, according to Nvidia, skipping TensorRT 7.0 for JetPack was a rare exception.
For now, we'll have to go with documenting TensorRT 6 using Onnxruntime 1.1.2 as you suggested.  
Thanks.

Reply:
Got it and I just made an update.